frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/edit.jsp

```diff
@@ -151,7 +151,7 @@
                                         <tbody>
                                             <core_rt:forEach items="${component.releases}" var="myRelease">
                                                 <tr>
-                                                    <td><a href="${myRelease.id}">${myRelease.name}</a></td>
+                                                    <td><a href='<portlet:renderURL><portlet:param name="<%=PortalConstants.COMPONENT_ID%>" value="${component.id}"/><portlet:param name="<%=PortalConstants.RELEASE_ID%>" value="${myRelease.id}"/><portlet:param name="<%=PortalConstants.PAGENAME%>" value="<%=PortalConstants.PAGENAME_EDIT_RELEASE%>"/></portlet:renderURL>'>${myRelease.name}</a></td>
```

Comment:
Please use `DisplayReleaseLink` tag.

Reply:
Using DisplayReleaseLink tag and hyperlink is added on version instead of Name as suggested